results/cmd/api/main_test.go

```diff
@@ -61,5 +63,40 @@ func setupTestDB(dbName string, t *testing.T) (*sql.DB, error) {
 	if _, err := statement.Exec(); err != nil {
 		t.Fatalf("failed to execute the taskrun table creation statement statement: %v", err)
 	}
-	return db, nil
+	return srv, nil
+}
+
+func TestDeleteTaskRun(t *testing.T) {
+	// Create a temporay database
+	srv, err := setupTestDB("testdb", t)
+	if err != nil {
+		t.Fatalf("failed to create temp file for db: %v", err)
+	}
+
+	// Connect to fake server and insert a new taskrun
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	r, err := srv.CreateTaskRun(ctx, &pb.CreateTaskRunRequest{TaskRun: &pb.TaskRun{
+		ApiVersion: "1",
+		Metadata: &pb.ObjectMeta{
+			Uid:       "123459",
+			Name:      "mytaskrun",
+			Namespace: "default"}}})
+	if err != nil {
+		t.Fatalf("could not create taskrun: %v", err)
+	}
+
+	// Delete inserted taskrun
```

Comment:
We should also test deleting a deleted taskrun.

Reply:
done